dask_ml/utils.py

```diff
@@ -246,6 +253,53 @@ def _format_bytes(n):
     return "%d B" % n
 
 
+@contextlib.contextmanager
+def _timer(name, _logger=None, level="info"):
+    """
+    Output execution time of a function to the given logger level
+
+    Parameters
+    ----------
+    name : str
+        How to name the timer (will be in the logs)
+    logger : logging.logger
+        The optional logger where to write
+    level : str
+        On which level to log the performance measurement
+    """
+    start = time.perf_counter()
```

Comment:
Looks like `time.perf_counter` is new in python 3.3, and we support 2.7.

Reply:
Yeah, I fixed that. I'm now checking (locally) why the `check_estimator` test is so slow.

Reply:
I've noticed some strangeness with that test previously. If necessary, we can disable it, but I'm a bit concerned that it's showing this change has a performance impact (though I don't see why that would be).

Reply:
@tomaugspurger I'm running it locally with the decorator turned off, but I don't see different results.

I'll test a bit more and report back.

Reply:
@TomAugspurger I forgot an indentation in the Lloyd loop :) Fixed now

Reply:
It's entirely possible that was my fault when I did the reformatting. If so then you have my apologies :)